From bfcf90c5a361acbb3b36f94ec73c9b67a4945471 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Tue, 21 Jun 2022 07:00:50 +0300
Subject: [PATCH] [SW-0] greco: fix size overflow issue

https://gerrit.habana-labs.com/#/c/222715/ patches
adds small common code. This causes .sram size overflow
ias below for greco

---------------x------------
region `.sram' overflowed by 8 bytes
---------------x------------

To quickly create some space rephrased print strings.

Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
Change-Id: I801803eb084ef7c6663eab73450c46ed8599704f
---

diff --git a/arch/greco/arc/common.c b/arch/greco/arc/common.c
index c4084f0..a095e5c 100644
--- a/arch/greco/arc/common.c
+++ b/arch/greco/arc/common.c
@@ -207,7 +207,7 @@
 		 * timeout and return
 		 */
 		if (timeout_expired(start, MEM_CPY_TIMEOUT_US)) {
-			preboot_puts("Timed-out waiting for mem_cpy\n");
+			preboot_puts("mem_cpy timedout\n");
 			break;
 		}
 	}
diff --git a/arch/greco/arc/security.c b/arch/greco/arc/security.c
index a24c37e..08c4ef7 100644
--- a/arch/greco/arc/security.c
+++ b/arch/greco/arc/security.c
@@ -54,12 +54,12 @@
 		return false;
 
 	if (pub_key_len != (KEY_SIZE - KEY_EXT_SIZE)) {
-		preboot_puts_hex("Invalid pub key length: ", pub_key_len, 0);
+		preboot_puts_hex("Invalid pub key len: ", pub_key_len, 0);
 		return false;
 	}
 
 	if (pub_key_ext_len != KEY_EXT_SIZE) {
-		preboot_puts_hex("Invalid pub key ext length: ",
+		preboot_puts_hex("Invalid pub key ext len: ",
 				pub_key_ext_len, 0);
 		return false;
 	}
@@ -94,7 +94,7 @@
 	sec_valid_fw_ver = SECURITY_FW_VERSION_TO_SECURITY_BYTE(valid_fw_ver);
 
 	if (sec_fw_ver < sec_valid_fw_ver)
-		preboot_puts_hex("Security version too low: ", sec_ver, 0);
+		preboot_puts_hex("Security version low: ", sec_ver, 0);
 
 	return sec_fw_ver >= sec_valid_fw_ver;
 }
